scss/components/_soe_news.scss

```diff
@@ -34,7 +34,7 @@
 
   .fullwidth .field-name-field-s-news-banner {
     .field-item {
-      background: $primary-color;
+      background: #f8f8f8;
```

Comment:
@mdyoung3,
If the banner background will be white, what's the purpose of this color change?

Reply:
@cjwest  The old banner was black.

Reply:
See the ticket.

Reply:
Got it!